tests/core/go_download_sdk/go_download_sdk_test.go

```diff
@@ -107,7 +107,7 @@ go_download_sdk(
 `,
 			optToWantVersion: map[string]string{
 				"": "go1.16",
-				"--@io_bazel_rules_go//go/toolchain:sdk_version=1":   "go1.16",
+				"--@io_bazel_rules_go//go/toolchain:sdk_version=1":      "go1.16",
```

Comment:
Looks like we are missing a test case for the new feature now.

Reply:
I'm not sure how to test that without depending on the host SDK. Any ideas?

Reply:
Good point. I think it's fine to test the `remote` case.